include/cipher.testsuite.testsuite.go.h

```diff
@@ -17,7 +15,7 @@
 #define JSON_FILE_SIZE      4096
 #define JSON_BIG_FILE_SIZE  32768
 
-// FIXME: Does not work on e.g. Windowa
+// FIXME: Does not work on e.g. Windows
```

Comment:
For the moment this will be tolerated ... full Windows support deferred until `0.26.0` .